etna/models/nn/chronos/base.py

```diff
@@ -0,0 +1,272 @@
+import os
+import warnings
+import zipfile
+from abc import abstractmethod
+from pathlib import Path
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Sequence
+from typing import Union
+from urllib import request
+
+import pandas as pd
+
+from etna import SETTINGS
+from etna.datasets import TSDataset
+from etna.distributions import BaseDistribution
+from etna.libs.chronos import BaseChronosPipeline
+from etna.libs.chronos import ChronosBoltModelForForecasting
+from etna.libs.chronos import ChronosModelForForecasting
+from etna.models.base import PredictionIntervalContextRequiredAbstractModel
+
+if SETTINGS.chronos_required:
+    import torch
+
+
+class ChronosBaseModel(PredictionIntervalContextRequiredAbstractModel):
+    """Base class for Chronos-like pretrained models."""
+
+    def __init__(
+        self,
+        model_name: str,
+        encoder_length: int,
+        device: str,
+        dtype: torch.dtype,
+        cache_dir: Path,
+        from_s3: bool,
+    ):
+        """
+        Init Chronos-like model.
+
+        Parameters
+        ----------
+        model_name:
+            Model name. See ``pretrained_model_name_or_path`` parameter of :py:func:`transformers.PreTrainedModel.from_pretrained`.
```

Comment:
It seems like it won't render properly in our documentation until we update `docs/soruce/conf.py` to index transformers documentation (I'm not sure if it is possible for non-Sphinx docs).

Reply:
We already have libs that render incorrect like `pytorch_forecasting`